docs/pages/connect-your-client/teleport-connect.mdx

```diff
@@ -97,18 +115,44 @@ cluster. This can help if you encounter a bug which renders the user interface p
 It might also help if you have issues with connecting to an active cluster that don't happen in the
 Web UI.
 
+<Tabs>
+<TabItem label="macOS">
 To force the app to log you out of all clusters, close the app and remove the `~/Library/Application
-Support/Teleport Connect/tsh` folder. Removing the file under `~/Library/Application
+Support/Teleport Connect/tsh` folder. Removing the file `~/Library/Application
 Support/Teleport Connect/app_state.json` will clear all remembered tabs and connections.
+</TabItem>
+<TabItem label="Linux">
+To force the app to log you out of all clusters, close the app and remove the `~/.config/Teleport
+Connect/tsh` folder. Removing the file `/.config/Teleport Connect/app_state.json` will clear
+all remembered tabs and connections.
+</TabItem>
+<TabItem label="Windows">
+To force the app to log you out of all clusters, close the app and remove the
+`C:\Users\yourUser\AppData\Roaming\Teleport Connect\tsh` folder. Removing the file
+`C:\Users\yourUser\AppData\Roaming\Teleport Connect\app_state.json` will clear all remembered tabs
+and connections.
+</TabItem>
+</Tabs>
 
 ### Submitting an issue
 
 To submit an issue, click the Submit Feedback button at the bottom right (the speech bubble symbol)
 and follow the *Submit a Bug* link.
 
-Be sure to attach logs which can be found under `~/Library/Application Support/Teleport
-Connect/logs`. The version of the app can be found in the app menu under the About Teleport Connect
-menu item.
+<Tabs>
+<TabItem label="macOS">
+Be sure to attach logs which can be found under `~/Library/Application Support/Teleport Connect/logs`.
+The version of the app can be found in the app menu under the About Teleport Connect menu item.
+</TabItem>
+<TabItem label="Linux">
+Be sure to attach logs which can be found under `~/.config/Teleport Connect/logs`. The app version
+can be found by pressing `Alt` to access the app menu -> Help -> About Teleport Connect
```

Comment:
Should we add a coma here?

```suggestion
can be found by pressing `Alt` to access the app menu -> Help -> About Teleport Connect.
```

Reply:
Couldn't hurt:

```suggestion
can be found by pressing `Alt` to access the app menu, then -> Help -> About Teleport Connect.
```

Reply:
Damn sorry, I meant a full stop not a comma, I often confuse these two in English for some reason. 😶 But I suppose it fits here!